### DISCUSSION TITLE
2023.9.17.0

### DISCUSSION THREAD

**Discussion by BornToBeRoot** [BornToBeRoot/NETworkManager#2499] [Releases]
[:page_facing_up: Changelog](https://github.com/BornToBeRoot/NETworkManager/blob/bdf646ac93a066f0e0b47ed24e5f8cc377537ba5/docs/Changelog/next-release.md)
[:book: Documentation](https://borntoberoot.net/NETworkManager/Documentation/Application)

⚠️⚠️⚠️
Due to significant refactoring, there may be various bugs in all tools. 
This pre-version may not be as stable as usual. 
Pease help testing and report bugs via github issues. 
⚠️⚠️⚠️

**Breaking Changes**
- Settings are reset due to the refactoring of the code base (Profiles are not affected)

**New Feature**
- Dashboard
  - Check IP geolocation
  - Check DNS resolver used
- Traceroute
  - Check IP geolocation

All features, improvements and bugfixes can be found in the changelog linked above!

---

| File | Checksum [SHA256]
|---|---|
|[Setup](https://github.com/BornToBeRoot/NETworkManager/releases/download/2023.9.17.0/NETworkManager_2023.9.17.0_Setup.exe)| `03CEA5F008AD9B30C56D2F45DDA2BB11F82DD9C7626D3CED416E44FA00118EAD` |
|[Portable](https://github.com/BornToBeRoot/NETworkManager/releases/download/2023.9.17.0/NETworkManager_2023.9.17.0_Portable.zip)| `1ED1F0E0E1B261C3DFCB0D2202B53F0C3531F37DCA455E7AF5E8BC87392192D4` |
|[Archive](https://github.com/BornToBeRoot/NETworkManager/releases/download/2023.9.17.0/NETworkManager_2023.9.17.0_Archive.zip)| `3E6F3C499E40102CC2D82F6888C60237882971739E4E99397BE11E1516D9FCF7` |

**System requirements**
- Windows 10 / Server x64 (1809 or later)
- [.NET Desktop Runtime 6.x (LTS)](https://dotnet.microsoft.com/download/dotnet/6.0)

<hr /><em>This discussion was created from the release <a href='https://github.com/BornToBeRoot/NETworkManager/releases/tag/2023.9.17.0'>2023.9.17.0</a>.</em>

**Comment by CH-SYR3** [comment]
I'm a noob: trying to install but all antivirus say I shouldn't... Is this safe to install?

**Reply by BornToBeRoot** [reply]
Hi @CH-SYR3,

The problem is the lack of reputation when I release new version (number of downloads etc.). I see this issue with Windows SmartScreen / Defender... You can wait a few days, then it should get whitelisted.

Or you can use an older version (e.g. the last release https://github.com/BornToBeRoot/NETworkManager/releases/tag/2023.6.27.1).

You can also check the binaries yourself on virustotal.com
Setup: https://www.virustotal.com/gui/file/03cea5f008ad9b30c56d2f45dda2bb11f82dd9c7626d3ced416e44fa00118ead

or compile them yourself #OpenSource :) 
See https://github.com/BornToBeRoot/NETworkManager#-build
